autobuild/android/autobuild_apk_1_prepare.sh

```diff
@@ -7,21 +7,17 @@
 ###  PROCEDURE  ###
 ###################
 
-#FROM ubuntu:18.04
-#based on https://gitlab.com/vikingsoftware/qt5.12.4androiddocker
-#LABEL "Maintainer"="Guenter Schwann"
-#LABEL "version"="0.3"
+COMMANDLINETOOLS_DIR=/opt/android/android-sdk/cmdline-tools/latest/
```

Comment:
Does this need to be called "latest", especially as we're pinning the version?

Reply:
Oh, I see - even the pinned file calls itself `_latest` 😄 ...

Reply:
I'm not sure. I've dropped the latest (and used even more existing variables instead of hardcoding path components). Let's see if it builds. I don't have any other way to check.

I guess the other instance (build-tools) needs to be latest as we don't pass that path anywhere.